packages/components/src/components/data-table/_data-table-core.scss

```diff
@@ -31,6 +31,10 @@
     padding: $spacing-05 0 $spacing-06 $spacing-05;
   }
 
+  .#{$prefix}--data-table-header--light {
+    background: $ui-02;
```

Comment:
can this explicitly target `background-color`?

Reply:
Definitely. This makes more sense in this case I guess!

Reply:
```suggestion
    background-color: $ui-02;
```

it looks like only L69 was changed to `background-color`

Reply:
Got it, sorry, @emyarod. Initially I thought you were referring to the color tokens (changing from `$ui-02` to `$ui-background`) to resolve the visual issue - hence the previous commits.

Should be all fixed now (back to `$ui-02` and targeting only the `background-color` instead of the `background` property) 🙂